src/node.cc

```diff
@@ -2681,6 +2681,9 @@ static void ProcessTitleSetter(Local<Name> property,
 static void EnvGetter(Local<Name> property,
                       const PropertyCallbackInfo<Value>& info) {
   Isolate* isolate = info.GetIsolate();
+  if (property->IsSymbol()) {
+    return info.GetReturnValue().SetUndefined();
```

Comment:
Isn't this the C++ equivalent of `function() { return undefined; }`? I've no strong feelings, but explicit isn't necessarily more readable, IMHO.